releasenotes/notes/add-utility-function-measure-4b951766cdfe5cd6.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Added a utility function - ``measure`` and ``measure_all`` in pulse module to measure qubits
+    using OpenPulse. For example::
```

Comment:
...in pulse module to return a ``Schedule`` which measures qubits using OpenPulse...